RequestGroup/RequestGroup-MedicationsPrescription-Example.json

```diff
@@ -0,0 +1,214 @@
+      "system": "https://fhir.nhs.uk/CodeSystem/EPS-task-business-status",
+      "code": "0003",
+      "display": "With Dispenser - Active"
+     }
+    }
+   ]
+  },
+  {
+   "url": "https://fhir.nhs.uk/StructureDefinition/Extension-PendingCancellation",
+   "valueBoolean": false
+  },
+  {
+   "url": "https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionType",
+   "valueCoding": {
+    "system": "https://fhir.nhs.uk/CodeSystem/prescription-type",
+    "code": "0101",
+    "display": "Primary Care Prescriber - Medical Prescriber"
+   }
+  }
+ ],
+ "identifier": [
+  {
+   "system": "https://fhir.nhs.uk/Id/prescription-order-number",
+   "value": "269CA9-A99968-4577BP"
+  }
+ ],
+ "status": "active",
+ "intent": "reflex-order",
+ "subject": {
+  "reference": "Patient/Patient-StaceyTwitchett-Example",
+  "identifier": {
+   "system": "https://fhir.nhs.uk/Id/nhs-number",
+   "value": "5839945242"
+  }
+ },
+ "authoredOn": "2025-01-29T12:00:00Z",
+ "author": {
+  "reference": "Organization/UKCore-Organization-HallgarthSurgery-Example",
+  "identifier": {
+   "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+   "value": "A83008"
+  }
+ },
+ "action": [
+  {
+   "title": "Prescription Line Items(Medications)",
+   "timingTiming": {
+    "event": [
+     "2025-01-29T12:00:00Z"
+    ],
+    "repeat": {
+     "frequency": 1,
+     "period": 20,
+     "periodUnit": "d"
+    }
+   },
+   "action": [
+    {
+     "id": "Medication-Number-1",
+     "resource": {
+      "reference": "MedicationRequest/MedicationRequest-Prescription-Example"
+     }
+    },
+    {
+     "id": "Medication-Number-2",
+     "resource": {
+      "reference": "MedicationRequest/MedicationRequest-RepeatPrescription-Example"
+     }
+    },
+    {
+     "id": "Medication-Number-3",
+     "resource": {
+      "reference": "MedicationRequest/MedicationRequest-PescriptionNumber3-Example"
+     }
+    },
+    {
+     "id": "Medication-Number-4",
+     "resource": {
+      "reference": "MedicationRequest/MedicationRequest-PescriptionNumber4-Example"
+     }
+    }
+   ]
+  },
+  {
+   "title": "Prescription status transitions",
+   "timingDateTime": "2025-01-29T13:00:00Z",
+   "action": [
+    {
+     "title": "Prescription upload successful",
+     "code": [
+      {
+       "coding": [
+        {
+         "system": "https://fhir.nhs.uk/CodeSystem/EPS-task-business-status",
+         "code": "0001",
+         "display": "To be Dispensed"
+        }
+       ]
+      }
+     ],
+     "participant": [
+      {
+       "extension": [
+        {
+         "url": "http://hl7.org/fhir/5.0/StructureDefinition/extension-RequestOrchestration.action.participant.typeReference",
+         "valueReference": {
+          "identifier": {
+           "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+           "value": "A83008"
+          }
+         }
+        }
+       ]
+      }
+     ],
+     "resource": {
+      "reference": "Task/Task-PrescriptionUpload-Example"
```

Comment:
Could this be a reference to the Message Bundle that created the Prescription instead? e.g. the one used in  EPS prescription-order?